src/main/java/seedu/address/logic/commands/SetLessonCommand.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.commands.RemoveClassCommand.MESSAGE_MISSING_CLASS_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLASS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LESSONQUANTITY;
+
+import java.util.List;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.module.Class;
+import seedu.address.model.module.ClassName;
+import seedu.address.model.module.exceptions.ClassNotFoundException;
+import seedu.address.model.student.Student;
+
+/**
+ * Sets the number of lessons of a Class.
+ */
+public class SetLessonCommand extends Command {
+    public static final String COMMAND_WORD = "setlesson /c";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sets the total number of lessons in the Class.\n"
+            + "Parameters: "
+            + PREFIX_CLASS + " CLASS_NAME"
+            + "\n"
+            + PREFIX_LESSONQUANTITY + " NUMBER_OF_LESSON"
+            + "\n"
+            + "Example: "
+            + COMMAND_WORD + " cs2103" + " /l 5";
+    public static final String MESSAGE_START_LESSON_SUCCESS = "Successfully set the number of lessons in %s to %d!";
+    private ClassName className;
+    private int numLessons;
+
+    /**
+     * Command to simulate starting a lesson of a Class thus increasing total lessons conducted of the Class by one.
```

Comment:
Code looks neat and clean! 
But the javadoc here seems to be for another command